### DISCUSSION TITLE
Hackathon Link

### DISCUSSION THREAD

**Discussion by Agrover112** [EddieHubCommunity/support#751] [General] [answered]
Sharing the link
https://ihack.ecell.in/#schedules

**Comment by Agrover112** [comment, marked as answer]
Link posted

**Comment by eddiejaoude** [comment]
It would be more useful if there was more information about the link and the event

**Reply by Agrover112** [reply]
I think all the information was mentioned at the bottom of the site

**Reply by Agrover112** [reply]
But I will make sure to leave a detailed description next time around

**Comment by vinzvinci** [comment]
## Important details.
![image](https://user-images.githubusercontent.com/73097560/106779664-2c4da600-6682-11eb-819f-728a4f0f7e9f.png)
